website/fb/sdoc-cache.json

```diff
@@ -0,0 +1,4 @@
+{
+  "snippets": {},
+  "description": "@generated"
+}
```

Comment:
This file is generated by docusaurus-plugin-internaldocs-fb, it should be checked in. It is used to store some dynamic content between the builds internally. Externally it won't change after the initial check in. Please do not add it to .gitignore file.

Reply:
Ok, but why isn't it checked in currently? 😅 

Should I add it as part of this PR?

Reply:
It's already checked in internally (ie, to our Sapling monorepo), but the `fb` directory is excluded from syncing with GitHub. It shouldn't be in the PR - just leave it unstaged or locally ignored.

(Tbh I'm not sure what would happen if I tried to import it 😅. I *guess* it would just be discarded.)